pkg/hubble/parser/getters/getters.go

```diff
@@ -40,6 +42,10 @@ type EndpointGetter interface {
 	GetEndpointInfoByID(id uint16) (endpoint v1.EndpointInfo, ok bool)
 }
 
+type EndpointsGetter interface {
+	GetEndpoints() map[policy.Endpoint]struct{}
+}
```

Comment:
Maybe worth documenting that this interface gets you `policy.Endpoint` rather than the "real" endpoints `endpoint.Endpoint`?

Reply:
Yup, forgot to add documentation.